penny_university/settings/base.py

```diff
@@ -169,6 +169,8 @@
 # Put product specific configuration below this.
 ############################################
 
+PENNYU_DOMAIN = 'pennyuniversity.org'
+
```

Comment:
Why is this necessary? Why not use FRONT_END_HOST?

Reply:
done